CONTRIBUTING.md

```diff
@@ -10,6 +10,59 @@ If you're interested in contributing documentation, please note the following:
 - Doc issues are raised in this repository, and they are tracked under the `kind/documentation` label.
 - Pull requests are submitted to the K3s documentation source in the [Rancher docs repository.](https://github.com/rancher/docs/) The K3s docs content is in the `content/k3s/` directory.
 
+## Pull request and git commit guidance
+
+### Opening PRs and organizing commits
+PRs should generally address only 1 issue at a time. If you need to fix two bugs, open two separate PRs. This will keep the scope of your pull requests smaller and allow them to be reviewed and merged more quickly.
+
+When possible, fill out as much detail in the pull request template as is reasonable. Most important is to reference the GitHub issue that you are addressing with the PR.
+
+**NOTE:** GitHub has [a feature](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) that will automatically close issues referenced with a keyword (such as "Fixes") by a PR or commit once the PR/commit is merged. Don't use these keywords. We don't want issues to be automatically closed. We want our testers to independently verify and close them.
```

Comment:
Great note. Curious if we can turn this off somehow or have a bot fix this for us by editing the PR

Reply:
Nobody else does this.... Which makes me think we are doing something wrong.

Reply:
It cannot be turned off in github (yet).

darren, what part do you think we're doing wrong? trying to keep issues open?

Reply:
Yeah, keeping issues open.  But it's going to go into a much, much bigger discussion. Probably shouldn't have said anything as this PR isn't the place to have that tangent.

Reply:
yep, no problem. i knew this would generate "fix the process" comments.